DRCDesignerLinux/rec_imp/rec_imp.cpp

```diff
@@ -21,9 +21,9 @@
 
 				d.sbragion@infotecna.it
 
-		This program uses the parsecfg library from Yuuki  NINOMIYA.  De�
-		tails  on  this  library  can be found in the parsecfg.c and par�
-		secfg.h files.  Many thanks to Yuuki NINOMIYA for this useful li�
+		This program uses the parsecfg library from Yuuki  NINOMIYA.  De�
```

Comment:
I cannot get rid of these changes. Perhaps someone can tell me what it is. I tried to set git to autoctrlf (false) but that didn't do the trick. I guess it has to do with windows encoding. I used Linux to change the file.